### PR TITLE
Update project

### PR DESCRIPTION
**Pull request by Mutalenic**
### In this `PR` I am doing the following:-

- Code-Reviewing
- Creating an issue

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.main-container{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+
```
Comment (Mutalenic) [inline styles.css:6]: The empty line(line 6) can be deleted to make the code look clean

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
+.main-container{
```
Comment (Mutalenic) [inline styles.css:1]: - In line 1 after .main-container, you can leave a space to make the code look better.
- Sep-up GitHub actions to address these issues

**Review by Mutalenic — COMMENTED** [review]
STATUS: Changes Required
Hello @harlexkhal ✋🏼
Your project is great. You are almost done. However, there are some issues you should fix. 

- Set-up linters or validators in your repo, check this link for more [information]( url : https://github.com/microverseinc/linters-config)
- Add a background color to make your image standout
- Add font color to make your website more eye-catching

**Review by VanessaAoki — APPROVED** [review]
## STATUS: APPROVED 🟢 
Hi @Mutalenic, 

Your project is complete!  There is nothing else to say other than... it's time to merge it! 
Congratulations! 🎉 

approved

### Project highlights:
- [x] The inline comments are polite and explain why you are requiring those changes, plus they are clear and easy to understand 🔥 
- [x] Great use of emojis, making it very human friendly, kudos 👽  


Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**